compiler/luci-interpreter/src/kernels/BinaryOpUtils.h

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
I think it will be better to name this file `BinaryOpCommon` or something, because this code is doing actual computations, not auxiliary actions.

Reply:
Fixed.